src/Level.cpp

```diff
@@ -174,5 +174,69 @@ void Level::setPlayerCellTransparency(const sf::Uint8& alpha)
     (*cells[playerIndex]).setCellColorTransparency(alpha);
 }
 
+/**
+ *
+ */
+void Level::movePlayerDown()
+{
+    /* reset the current player cell transparency */
+    setPlayerCellTransparency(255);
+
+    playerIndex += 20;
+
+    /* show the player cell */
+    showPlayerCell();
+}
+
+/**
+ *
+ */
+void Level::movePlayerUp()
+{
+    /* reset the current player cell transparency */
+    setPlayerCellTransparency(255);
+
+    playerIndex -= 20;
+
+    /* show the player cell */
+    showPlayerCell();
+}
+
+/**
+ *
+ */
+void Level::movePlayerLeft()
+{
+    /* reset the current player cell transparency */
+    setPlayerCellTransparency(255);
+
+    playerIndex -= 1;
+
+    /* show the player cell */
+    showPlayerCell();
+}
+
+/**
+ *
+ */
+void Level::movePlayerRight()
+{
+    /* reset the current player cell transparency */
+    setPlayerCellTransparency(255);
+
+    playerIndex += 1;
+
+    /* show the player cell */
+    showPlayerCell();
```

Comment:
move setPlayerCellTransparency and showPLayerCell to call them all the time automatically